### PR TITLE
Move OFF 7.2.3 IVS to annex

### PR DESCRIPTION
**Pull request by PeterConstable**
Move section with only one small edit (add missing ":" on last line)

### CODE REVIEW

File: colr-gradients-spec.md [hunk 1]
```diff
@@ -1329,6 +1200,135 @@ _Replace the table defining data types with the following (added row for Offset2
...
+
+_Insert a sub-clause heading, "7.2.3.2 Variation data", after the newly-inserted text above, and before the paragraph beginning, "The ItemVariationStore table includes a variation region list..." Re-number subsequent sub-clauses accordingly._
+
+_In the fifth paragraph that follows the figure in (now) 7.2.3.2, delete the first sentence, "A complete delta set index... within that subtable." Before that pragraph, insert the following paragraph:_
+
+A complete delta-set index involves an outer-level index into the ItemVariationData subtable array, plus an inner-level index to a delta-set row within that subtable. A special meaning is assigned to a delta-set index 0xFFFF/0xFFFF (that is, outer-level and inner-level portions are both 0xFFFF): this is used to indicate that there is no variation data for a given item. Functionally, this would be equivalent to referencing delta-set data consisting of only deltas of 0 for all regions.
+
+_In 7.2.3.3 (previously, 7.2.3.1), "Variation regions", in the table for the VariationRegionList structure, add the following sentence to the end of the description for the regionCount field._
+
+Shall be less than 32,736.
```
Comment (drott) [inline colr-gradients-spec.md:1278]: Move itself is fine with me.

```suggestion
Must be less than 32,736.
```

This is a change from the previous text. For tracking changes, let's really have isolated move diffs that do not do any edits, then do changes in separate CLs.

Comment (PeterConstable) [reply colr-gradients-spec.md:1278]: Unintentional. Probably because of #116 that was merged while I had parallel changes, and not fully merged. Will fix.

Comment (PeterConstable) [reply colr-gradients-spec.md]: Changed

**Review by drott — APPROVED** [review]
Thanks, LGTM.

**Pull request merged**